plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -353,9 +636,11 @@ fu_cros_ec_usb_device_class_init (FuCrosEcUsbDeviceClass *klass)
 {
 	FuDeviceClass *klass_device = FU_DEVICE_CLASS (klass);
 	FuUsbDeviceClass *klass_usb_device = FU_USB_DEVICE_CLASS (klass);
+	klass_device->write_firmware = fu_cros_ec_usb_device_write_firmware;
 	klass_device->setup = fu_cros_ec_usb_device_setup;
 	klass_device->to_string = fu_cros_ec_usb_device_to_string;
 	klass_usb_device->open = fu_cros_ec_usb_device_open;
 	klass_usb_device->probe = fu_cros_ec_usb_device_probe;
 	klass_usb_device->close = fu_cros_ec_usb_device_close;
+	klass_device->prepare_firmware = fu_cros_ec_usb_device_prepare_firmware;
```

Comment:
a total nit, but please order the `klass_device` stuff together

Reply:
Done.